src/nvme/mi.h

```diff
@@ -367,6 +367,34 @@ struct nvme_mi_admin_resp_hdr {
 	__le32	cdw0, cdw1, cdw3;
 } __attribute__((packed));
 
+/**
+ * struct nvme_mi_control_req - The Control Primitive request.
+ * @hdr: Generic MI message header
+ * @opcode: Control Primitive Opcodes (using &enum nvme_control_opcode)
```

Comment:
Don't forget to replace this `nvme_control_opcode` reference if you rename the enum.

Reply:
yup